contracts/interfaces/IAMPLGovernance.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity 0.4.24;
```

Comment:
I think it's ok if the governance contracts simply declare the interface that's needed in the files where it's used. (Actually, I don't think they'd actually be used anyway since they'd be called in a generalized fashion from the voting contract). 

So I don't think we need this file.